src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -377,8 +377,21 @@ ReportClientSideStats(
 
   std::cout << "    Request count: " << stats.request_count << std::endl;
   if (stats.delayed_request_count != 0) {
-    std::cout << "    Delayed Request Count: " << stats.delayed_request_count
-              << std::endl;
+    std::stringstream delay_data{""};
+    delay_data << "    Delayed Request Count: " << stats.delayed_request_count
+               << std::endl;
+    delay_data << "    "
+               << "Avg send request rate: " << std::fixed
+               << std::setprecision(2) << send_request_rate << " infer/sec"
+               << std::endl;
+    delay_data << "    "
+               << "[WARNING] Perf Analyzer is not able to keep up with the "
+                  "desired load. ";
+    float delay_pct =
+        ((float)stats.delayed_request_count / stats.request_count) * 100;
+    delay_data << delay_pct << "% of the requests were delayed. ";
```

Comment:
My concern is that load/delayed might not make full sense to a random user. Does this read a little cleaner?

"Perf Analyzer was not able to keep up with the desired request rate. X% of the requests were delayed"

Reply:
I like that more. Updated.